pywinauto/mouse.py

```diff
@@ -238,8 +239,24 @@ def right_click(coords=(0, 0)):
     _perform_click_input(button='right', coords=coords)
 
 
-def move(coords=(0, 0)):
+def move(coords=(0, 0), duration=0):
     """Move the mouse"""
+    minimum_duration = 0.05
+    if duration >= minimum_duration:
+        coord_start = win32api.GetCursorPos()
```

Comment:
I'd suggest `x_start, y_start = win32api.GetCursorPos()` to avoid some magic numbers 0 and 1.

Reply:
done, b7efe5fb7acae1ec3caa81c3b256ba9d1f23f6d6